### DISCUSSION TITLE
Sponsor and Support

### DISCUSSION THREAD

**Discussion by tarekmaalouf** [headtracker/HeadTracker#97] [Q&A]
Just for the creators, I want you to know that I want to sponsor, support, and fund the project and other projects you may be working on. I see real talent here and am so impressed with the overall work you have put into this project. I would like to have a discussion about this if and when you may be interested. Thank you.

**Comment by dlktdr** [comment]
I have a plan to build a custom board when the chip I would like to use is back in stock to help with some of the downfalls of the Nano BLE 33. Although it's okay there is room for hardware improvement IMO. Contact me on discord if you want to chat more. https://discord.gg/ux5hEaNSPQ  PM - @Cliff(dlktdr)
